buildSrc/src/main/groovy/com/linkedin/gradle/build/DistributeTask.groovy

```diff
@@ -9,46 +9,46 @@ import org.jfrog.gradle.plugin.artifactory.dsl.ArtifactoryPluginConvention
 
 class DistributeTask extends DefaultTask {
```

Comment:
I would not roll out buildSrc for just one class. I would drop it in a separate gradle file and import the file using "apply from:". However, it's a nitpick your code works ;)

DistributeTask.groovy is Groovy code but has java artifacts like semicolons

Reply:
removed the semi colons. i'd be happiest to not have this at all, but that depends on https://github.com/JFrogDev/build-info/pull/128 happening.